src/steps/command.ts

```diff
@@ -28,26 +28,34 @@ function assertSkipValue(value: SkipValue): void {
 type SkipValue = boolean | string;
 export type SkipFunction = () => SkipValue;
 
+let transformCommandKey = Symbol();
```

Comment:
```suggestion
const transformCommandKey = Symbol();
```

Reply:
this currently fails due to: https://github.com/Microsoft/TypeScript/issues/24587
It seems a fix landed in https://github.com/microsoft/TypeScript/pull/44512 today but hasn't been released, yet. There is a workaround by casting the symbol...

```suggestion
const transformCommandKey: string = Symbol('transformCommand') as any;
```